### PR TITLE
US038-adding-linting-checks

### PR DESCRIPTION
**Pull request by Cormac-F**
- Installed ESLint and the relevant typescript dependencies
- Created YAML file that checks the changed files and lints them

Test file - 
Screenshot 2023-12-05 at 3 52 06 pm

Output on the workflow build -
Screenshot 2023-12-05 at 3 51 33 pm

Output when pull request is created -
Screenshot 2023-12-05 at 3 51 51 pm

*PLEASE NOTE*
~~ESLint doesn't work locally due to the version of node being used (v14)~~
ESLint now works locally - run the following commands
nvm install 16
nvm use 16
npm run lint
GitHub Action builds using v16 to avoid multiple errors that have been noted

### CODE REVIEW

File: .gitignore
Comment (jkelso662) [inline .gitignore:1290]: I think L1290 is caught by `dist/` and L1288 is caught by `node_modules/` presumably

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Looks good! Maybe worth checking in with team 4 regarding getting eslint running locally. Would also like to hear if you do find out anything useful.

Noticed your PR title is missing part of the ticket code. Also you need to add Joanne as an approver as well as me. Let me know when you are ready for another review by getting github to request another review of me and Joanne. :)

**Comment by jkelso662** [comment]
One more thing, noticed this ticket was in the backlog thismorning. If it is being worked on, it should be `In Progress`, or if it is waiting for code review it should be in `In Review`

**Comment by Cormac-F** [comment]
- Updated ESLint file
- Found local fix by using nvm to change node version to 16
- Updated PR name
- Updated git ignore

Working locally -
Screenshot 2023-12-06 at 11 20 35 am

**Review by jkelso662 — APPROVED** [review]
Looks good. Fab finding the fix. Better to update the description to say about eslint working locally.

**Review by CharlesM988 — APPROVED** [review]
Looks great!

**Pull request merged**
